tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/MetadataOperations.java

```diff
@@ -163,6 +167,13 @@ public boolean typeWithDeclaredSubtypesMetadata(Shape plainShape, Rectangle rect
     return false;
   }
 
+  @Query(translator = MetadataExtensionQueryTranslator.class,
+      entityResolver = MetadataExtensionEntityResolver.class,
+      outputResolver = NativeQueryOutputResolver.class)
```

Comment:
`outputResolver` may makes you think that is "THE" resolver, meanwhile the metadata can be resolved with the entities or the native query. `nativeResolver` could be a better name.